workflow/common/common.go

```diff
@@ -15,6 +15,9 @@ const (
 	// DockerSockVolumeName is the volume name for the /var/run/docker.sock host path volume
 	DockerSockVolumeName = "docker-sock"
 
+	// AnnotationKeyDefaultContainer is the annotation that specify container that will be used by default in case of kubectl commands for example
+	AnnotationKeyDefaultContainer = "kubectl.kubernetes.io/default-container"
```

Comment:
According to the k8s docs [kubectl.kubernetes.io/default-container](https://kubernetes.io/docs/reference/labels-annotations-taints/#kubectl-kubernetes-io-default-container) is the annotation, not a label.